libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAbstractPasscodeViewController.m

```diff
@@ -105,31 +104,34 @@ - (BOOL)decrementPasscodeAttempts
 - (void)validatePasscodeFailed
 {
     dispatch_async(dispatch_get_main_queue(), ^{
-        self.remainingAttempts = kMaxNumberofAttempts;
+        self.remainingAttempts = (_viewConfig.maxNumberOfAttempts) ? _viewConfig.maxNumberOfAttempts : kMaxNumberofAttempts;
```

Comment:
Why not do self.viewConfig instead of _viewConfig?

Reply:
Also the code 
```obj-c
self.remainingAttempts = (_viewConfig.maxNumberOfAttempts) ? _viewConfig.maxNumberOfAttempts : kMaxNumberofAttempts;
```
 is in multiple place. Should we move it to a helper method (something like `updateRemainingAttempts`) or maybe have
```obj-c 
- (NSInteger) remainingAttempts {
  return  (_viewConfig.maxNumberOfAttempts) ? _viewConfig.maxNumberOfAttempts : kMaxNumberofAttempts;
}
```